### PR TITLE
normalize os.homedir() and os.tmpdir()

### PR DESCRIPTION
**Pull request by chrisblossom**

### CODE REVIEW

**Comment by chrisblossom** [comment]
Test looks to be failing on AppVeyor because the temp directory is inside the home directory. It is late so I'll look how to solve this later. If you have any thoughts please let me know.

EDIT: I swapped the order of ``<HOME_DIR>`` and ``<TEMP_DIR>`` and the tests pass now. I will add some more tests to verify this what is the expected result later.

**Comment by tribou** [comment]
Awesome, good catch and let me know when this is ready.

**Comment by chrisblossom** [comment]
Tested [``<HOME_DIR>`` inside ``<TEMP_DIR>``](https://github.com/tribou/jest-serializer-path/pull/17/files#diff-044bc223cd3309d59e7054541146894aR329) and [``<TEMP_DIR>`` inside ``<HOME_DIR>``](https://github.com/tribou/jest-serializer-path/pull/17/files#diff-044bc223cd3309d59e7054541146894aR312).

Ready to be merged.

File: README.md [hunk 1]
```diff
@@ -40,6 +37,15 @@ All absolute paths will now be converted and saved in snapshots like so:
...
 
+``/path/to/os-temp/nested/temp`` => ``<TEMP_DIR>/nested/temp``
+
+``/path/to/user-home/nested/home`` => ``<HOME_DIR>/nested/home``
+
+#### Caveats
+
+* All single backslashes (`\`) will be replaced by a forward slash (`/`)
+* Any string that looks like a Windows drive letter (`C:\`) will be replaced by a forward slash (`/`).
+
```
Comment (chrisblossom) [inline README.md]: I moved the note because I had added another, but later removed/solved it. Thought it was better left like this.

**Comment by tribou** [comment]
Nice work adding those tests. Thanks again!

**Pull request merged**